sos/report/plugins/postfix.py

```diff
@@ -52,6 +52,31 @@ def forbidden_ssl_keys_files(self):
         finally:
             return fp
 
+    def forbidden_password_files(self):
+        forbid_attributes = [
+            "lmtp_sasl_password_maps",
+            "smtp_sasl_password_maps",
+            "postscreen_dnsbl_reply_map",
+            "smtp_sasl_auth_cache_name",
+        ]
+        fp = []
+        prefix = 'hash:'
+        try:
+            with open(self.path_join('/etc/postfix/main.cf'), 'r') as cffile:
+                for line in cffile.readlines():
+                    # ignore comments and take the first word after '='
+                    if line.startswith('#'):
+                        continue
+                    words = line.split('=')
+                    if words[0].strip() in forbid_attributes:
+                        filepath = words[1].split()[0]
```

Comment:
Assume a corner case:

    lmtp_sasl_password_maps = 

which would raise `IndexError: list index out of range` error ;-)

What about:

    words = line.replace(' ', '').split('=')
    if words[0] in forbid_attributes and len(words) > 1:
        filepath = words[1]

(this won't work if the file path contains space - is that syntax allowed?)